src/components/workspace/MCQChooser.tsx

```diff
@@ -33,6 +48,22 @@ class MCQChooser extends React.PureComponent<IMCQChooserProps, {}> {
       </div>
     )
   }
+
+  /**
+   * A function to generate an onClick function that causes
+   * and mcq submission with a given answer id.
+   *
+   * Post-condition: the local state will be updated to store the
+   * mcq option selected.
+   *
+   * @param i the id of the answer
+   */
+  private onButtonClickFactory = (i: number) => (e: any) => {
```

Comment:
Should I stick to this or make a tslint exception?

Reply:
I think it's ok